src/config.rs

```diff
@@ -227,6 +257,10 @@ impl Config {
             })
         }
     }
+
+    pub fn index_settings(&self) -> Bytes {
+        bincode::serialize(&(self.txstore_enabled, self.blocktxs_enabled, self.blockmeta_enabled)).unwrap()
```

Comment:
Maybe we can encapsulate the settings that require reindexing in a sub-struct under `Config`?
This way we can even serialize it to JSON using serde, and store the JSON in RocksDB.
(later we can add a "DB version" field, to force re-indexing when the semantics of some existing field changes. similar to https://github.com/trezor/blockbook/blame/master/docs/rocksdb.md#L27)

Reply:
If it's fine by you, I prefer using JSON, since it adds a name to each value (helping us to prevent re-ordering errors, and making the debugging easier).

Reply:
After discussion, we ended up using `(network_type,db_version,extended_db_enabled)` for the compatibility field.